agent/stats/utils_windows.go

```diff
@@ -43,3 +43,7 @@ func validateDockerStats(dockerStats *types.StatsJSON) error {
 	}
 	return nil
 }
+
+func getStatusMessageReportingPath(dataDirOnHost, taskARN, containerName string) (string, error) {
+	return "", nil
```

Comment:
Maybe add a comment here about windows is not supported to avoid confusion.

Reply:
Done:)